src/pages/hegel/reference/nothing/difference-between-being-and-nothing.mdx

```diff
@@ -0,0 +1,202 @@
+distinctions between `being` and `nothing`, such as that found in Parmenides
+which resists the thought that they could ever be the same (Houlgate 2022, 146).
+The idea that a thing could only ever be the thing that it is, or whose being is
+identical to itself, is disproven at the very outset of the _Logic_, for at the
+very least `being` (and `nothing`) is unable to _simply be_ &ndash; it vanishes
+into `nothing`.
+
+As Houlgate describes it, there is nothing mysterious or irrational about the
+dialectical change taking place in `being` and `nothing` into each other. "That
+dialectical conversion is logically necessary: being and nothing pass into one
+another for the reasons we have seen, and the immediate difference between them
+thereby proves to be, of necessity, an indeterminate, unstable difference"
+(Houlgate 2022, 146). Hegel therefore does not mean to flout the traditional
+principles of reasoning or to undermine rational argument, he is simply
+revealing the "dynamism in being that he takes to become evident when one
+focuses on being in its purity and immediacy _without uncritically assumed
+preconceptions_" (Houlgate 2022, 146). When `being` is considered
+[without presuppositions](/hegel/guides/presuppositionless-thinking), the
+argument of the _Logic_ is unassailable: `pure being` is utterly incapable to
+simply be.
+
+### Parallel Categories, Divergent Conceptions (Niklas)
+
+When regarding the respective transitions of `being` and `nothing`, note that in
+the text concerning the development of these categories is slightly divergent.
+The development of `being` ends with it turning out to be `nothing`, whereas the
+development of `nothing` ends with it turning out to be _the same as_ `being`.
+
+The former case uses _is_ whereas the latter case uses _same_. The first
+signifies a more immediate transition whereas the latter seems to convey the
+sense of some essence, an "area" where being and nothing are conjoined. If the
+two categories were to be differentiated in terms of immediacy and mediation,
+`nothing` appears to fall on the side of mediation. If this notion of _sameness_
+indeed belongs to `nothing` inasmuch as it is the thought that reveals the
+sameness of the two, then it could be thought of as the "gathering" or
+aggregating category whilst `being` signals "dispersal" or diremption.
+
+This has the implication that not only does `being` and `nothing` immediately
+mean something different, but that there is a different conceptual movement at
+work in each of them. One breaks off into new ground whereas the other coalesces
+a return to what was.
+
+Alternatively, this element of sameness is conjoined with the intention of the
+immediate difference between `being` and `nothing`, and that that intention is
+the presence of `becoming` (Hegel 2010, 68/21.79), such that this is not a
+peculiar property of `being` or `nothing` but is the alpha of the emergent
+category.
```

Comment:
I think you're definitely onto something. And I think what you're pointing at is the fact that there actually is some determination and the development is not as indeterminate as Hegelgate suggest.